docs/src/references/c8y-configuration-management.md

```diff
@@ -78,6 +78,8 @@ Each configuration file is defined by a record with:
   These are only used when a configuration file pushed from the cloud doesn't exist on the device.
   When a configuration file is already present on the device, this plugin never changes file ownership,
   ignoring these parameters.
+* Optional `childid`. For details see section "Configuration files for child devices" below.
```

Comment:
Not quite convinced with this idea of having the configs of the root device and the child devices in the same config file, from a UX point-of-view. The whole idea of having a child device as a dedicated device twin was to ease the management of everything related to that child devices without any interference from other devices. By mixing things up, we're going against that fundamental principle. With this UX, whenever I want to manage the config file list of a child device from the cloud, the user will always have to navigate away from the child device twin to the thin-edge root device twin. That's not a very pleasant UX in my view.

Reply:
I fully agree with @albinsuresh. We need to consider the alternative of one `c8y_configuration_plugin.toml` file per child device.